website/content/en/docs/helm/quickstart.md

```diff
@@ -231,83 +230,113 @@ using a bit of setup from other `operator-sdk` commands. Check out the OLM integ
 Apply the nginx CR that we modified earlier:
 
 ```sh
-kubectl apply -f deploy/crds/example.com_v1alpha1_nginx_cr.yaml
+$ kubectl apply -f config/samples/example_v1alpha1_nginx.yaml -n nginx-operator-system
```

Comment:
The operator is cluster-scoped out of the box, so let's create the CR in the namespace that the user has configured in their current context.
```suggestion
$ kubectl apply -f config/samples/example_v1alpha1_nginx.yaml
```